pkg/webhook/namespace/utils.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020-2023 Project Capsule Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package namespace
+
+import (
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	capsulev1beta2 "github.com/projectcapsule/capsule/api/v1beta2"
+)
+
+const (
+	ObjectReferenceTenantKind = "Tenant"
+)
+
+func isTenantOwnerReference(or metav1.OwnerReference) bool {
+	parts := strings.Split(or.APIVersion, "/")
+	if len(parts) != 2 {
+		return false
+	}
+
+	group := parts[0]
+	version := parts[1]
+
+	return group == capsulev1beta2.GroupVersion.Group && version == capsulev1beta2.GroupVersion.Version
```

Comment:
Could be this a problem with previous Capsule versions? I'm referring to the `Version`.

Reply:
Yeah, we can also only check the Group

Reply:
Changed the check, verify against group and kind. Disregarding version